api/appwrite.ts

```diff
@@ -0,0 +1,13 @@
+// appwrite.ts
+
+import { Client, Databases, Account } from "appwrite";
+
+const url: string = import.meta.env.VITE_APPWRITE_ENDPOINT;
```

Comment:
So, we're not using environment variables yet. This should be hard coded with our appwrite URL.